### PR TITLE
rework description

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

**Comment by r10s** [comment]
lgtm, great. tons of times better than the old one :)

File: store/key-features.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+Key features at a glance:
+
+* Automatic e-mail end-to-end-encryption with Autocrypt
+* Instant chatting and synchronization across devices
+* Reach anyone out of billions of people -- just use their e-mail address!
+* Open Source developments with many contributors and a diverse community
```
Comment (LenzGr) [inline store/key-features.txt]:
```suggestion
* Open Source development with many contributors and a diverse community
```

File: store/key-features.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+* No Spam -- only messages of known users are shown by default
```
Comment (LenzGr) [inline store/key-features.txt]:
```suggestion
* No Spam -- only messages of known users are displayed by default
```

File: store/key-features.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+Key features at a glance:
+
+* Automatic e-mail end-to-end-encryption with Autocrypt
+* Instant chatting and synchronization across devices
+* Reach anyone out of billions of people -- just use their e-mail address!
```
Comment (LenzGr) [inline store/key-features.txt]: I'd actually move this aspect to the top.

File: store/key-features.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+Key features at a glance:
+
+* Automatic e-mail end-to-end-encryption with Autocrypt
```
Comment (LenzGr) [inline store/key-features.txt]:
```suggestion
* Automatic end-to-end-encryption based on Autocrypt
```

File: store/key-features.txt (hunk 3 above)
Comment (LenzGr) [reply store/key-features.txt]: Any how about this?
```suggestion
* Chat with billions of people -- using their e-mail address!
```

File: store/key-features.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+* Instant chatting and synchronization across devices
+* Reach anyone out of billions of people -- just use their e-mail address!
+* Open Source developments with many contributors and a diverse community
+* No Spam -- only messages of known users are shown by default
+* Fully Standards based, collaborations ongoing with many other projects
```
Comment (LenzGr) [inline store/key-features.txt]:
```suggestion
* Based on open standards, in collaboration with many other projects
```

File: store/key-features.txt (hunk 3 above)
Comment (hpk42) [reply store/key-features.txt]: i had this first -- but do you want to chat with billions? ;)

File: store/key-features.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+
+Key features at a glance:
+
+* Reach anyone out of billions of people -- just use their e-mail address!
+* Instant chatting and synchronization across devices
+* Automatic e-mail end-to-end-encryption with Autocrypt
+* available for Android, iOS, Mac OS, Linux with Windows in the works.
```
Comment (LenzGr) [inline store/key-features.txt]:
```suggestion
* Available for Android, iOS and Mac OS (Linux and Windows in the works)
```

File: store/key-features.txt (hunk 3 above)
Comment (LenzGr) [reply store/key-features.txt]: How about this?
```suggestion
* Chat with anyone, using their e-mail address
```

File: store/text.md [hunk 7]
```diff
@@ -1,29 +1,59 @@
...
-* Largest userbase -- receivers not using Delta Chat can be reached as well
-* Compatible -- not only to itself
-* Elegant and simple user interface
-* Distributed system
-* No Spam -- only messages of known users are shown by default
-* Reliable -- safe for professional use
-* Trustworthy -- can even be used for business messages
-* Fully OpenSource and Standards based
-
+Delta Chat feels like Whatsapp or Telegram but does not involve centralized control
```
Comment (LenzGr) [inline store/text.md]:
```suggestion
Delta Chat looks and feels like other popular messenger apps, but does not involve centralized control
```

File: store/text.md [hunk 8]
```diff
@@ -1,29 +1,59 @@
...
+or tracking. How does it achieve that? Delta Chat is technically an e-mail application
```
Comment (LenzGr) [inline store/text.md]:
```suggestion
or tracking. Technically, Delta Chat is an e-mail application with a modern chat interface.
```

**Comment by angelo-fuchs** [comment]
My latest change aimed at removing the "Silo", the ads element is purely optional.
Other lines that would do just as well:
> Chat over e-mail. Simple and Secured. No tracking, no ads: Privacy by default

> Chat over e-mail. Simple and Secured. Privacy by default

> Chat over e-mail. Simple and Secure. Your control: Privacy by default.

**Comment by r10s** [comment]
i personally would vote for the "No silos ... Privacy by design." thingie.
might be that it is not widely used, but for the ppl we target with this beta it might work well.

**Comment by LenzGr** [comment]
Making changes to a longer text in a github PR is somewhat cumbersome. I tried to slightly improve/polish the `text.md` file here: https://cryptpad.fr/code/#/2/code/edit/RgEQkbXeYvfUUBY+Si5mmP9D/

**Comment by angelo-fuchs** [comment]
lgtm. Minor thing about the short: Change the last ',' to 'and'?

> Chat over e-mail. Elegant, encrypted and without tracking: Privacy by Design.

**Comment by r10s** [comment]
okay, i think we can merge and upload it to gplay.
things can be refined later any time.

one question: what to do with the key-features.txt file? there is no such entry field available in gplay.

shall we add it to the end of the description? or just keep it for later user?

**Comment by r10s** [comment]
i've also added a pr to deltachat-pages that adds the links to gplay:  
https://github.com/deltachat/deltachat-pages/pull/135

page preview is available at:  
https://deploy-preview-135--deltachat.netlify.com/en/download

**Comment by r10s** [comment]
merged and uploaded to the developer console, once "reviewed" by gplay, it will be at https://play.google.com/store/apps/details?id=chat.delta

**Pull request merged**